readme.md

```diff
@@ -20,8 +20,8 @@ Create a pages with product card using `flexbox`, `BEM` and `SCSS` based on [thi
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+- [DEMO LINK](https://<RPrykhodko>.github.io/layout_product-cards/)
+- [TEST REPORT LINK](https://<RPrykhodko>.github.io/layout_product-cards/report/html_report/)
```

Comment:
```suggestion
- [DEMO LINK](https://RPrykhodko.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://RPrykhodko.github.io/layout_product-cards/report/html_report/)
```
fix your links firstly and try to re-deploy the work after that

Reply:
sure) thanks